test/integration/targets/docker_prune/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Create random names
+  set_fact:
+    cname: "{{ 'ansible-container-%0x' % ((2**32) | random) }}"
+    nname: "{{ 'ansible-network-%0x' % ((2**32) | random) }}"
+    vname: "{{ 'ansible-volume-%0x' % ((2**32) | random) }}"
+
+- block:
+  # Create objects to be pruned
+  - docker_container:
+      name: "{{ cname }}"
+      image: hello-world
```

Comment:
Is this going to run into the same issue as the idempotency fix? i.e. Does it need to be a running container?

Reply:
No, it actually needs to be *not* running, as it otherwise won't get pruned :)